guides/v2.0/cloud/live/stage-prod-migrate.md

```diff
@@ -117,15 +117,13 @@ For additional options, see the [rsync man page](http://linux.die.net/man/1/rsyn
 
 To migrate static files:
 
-1.	Open an SSH connection to your Staging or Production environment:
+1.	Copy media folder to your local environment:
```

Comment:
I think we should recommend using ssh agent forwarding instead.  As other said, it is a lot quicker, and easier.  We just need to explain how to do it (or link to third party documentation on how to do it).